### ISSUE TITLE
internal\certs\certs.go:348:12: nilness: tautological condition: nil == nil (govet)

### ISSUE THREAD

**Issue by atc0005** [atc0005/check-cert#816]
Linting error raised by govet:

```log
internal\certs\certs.go:348:12: nilness: tautological condition: nil == nil (govet)
                if block == nil && len(rest) > 0 {
```

This was also flagged by a recent `gopls` release.
